launcher/cog-launcher.c

```diff
@@ -267,6 +267,29 @@ on_automation_started(WebKitWebContext *context, WebKitAutomationSession *sessio
     g_signal_connect(session, "create-web-view", G_CALLBACK(on_automation_session_create_web_view), launcher);
 }
 
+static void
+cog_launcher_set_preferred_languages(CogLauncher *self)
+{
+    const gchar *const *locales;
+    const gchar       **languages;
+    guint               i, n, length;
+
+    /* build Accept-Language from locale, filtering out encodings */
+    locales = g_get_language_names();
+    length = g_strv_length((gchar **) g_get_language_names());
```

Comment:
I think we might have to reformat the languages.

MDN says the format is `en-US` yet this will return `en_US`.

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept-Language